GitUp/Application/AppDelegate.m

```diff
@@ -560,7 +560,7 @@ - (IBAction)newRepository:(id)sender {
   if ([savePanel runModal] == NSFileHandlingPanelOKButton) {
     NSString* path = savePanel.URL.path;
     NSError* error;
-    if (![[NSFileManager defaultManager] fileExistsAtPath:path] || [[NSFileManager defaultManager] moveItemAtPathToTrash:path error:&error]) {
+    if (![[NSFileManager defaultManager] fileExistsAtPath:path followSymlink:NO] || [[NSFileManager defaultManager] moveItemAtPathToTrash:path error:&error]) {
```

Comment:
Fair enough. So we need `-moveItemAtPath...` which either follows all symlinks or follows no symlinks.